package.json

```diff
@@ -17,7 +17,6 @@
   "license": "BSD-2-Clause",
   "devDependencies": {
     "mocha": "2.5.3",
-    "phantomjs": "1.9.20",
```

Comment:
What's the story here?

Reply:
I just wanted to test it out and see if anything else broke. Do we actually use this anywhere?

Reply:
It's almost certainly used by Mocha when running unit tests, but if things aren't breaking perhaps it's included in there as a peer dependency?

Reply:
How about a compromise: we keep phantomjs but upgrade to prevent strange testing failures?

Reply:
It's a deal 🤝.